src/assets/sass/main.scss

```diff
@@ -34,6 +34,7 @@ $button-padding-horizontal: 0.75em;
 @import '~bulma/sass/components/dropdown.sass';
 @import '~bulma/sass/components/level.sass';
 @import '~bulma/sass/components/modal.sass';
+@import '~bulma/sass/components/tabs.sass';
```

Comment:
Could you check what is the weight delta of the final bundle with this change ? (`npm run build` will give you the size of the bundles)

Our bundles are pretty big, and we must be carefull with this. As the component is used only once, for two tabs, it may worth a quick hack if it's too heavy.

Reply:
Pretty insignificant. Bundle goes from 33.62Ko to 33.99Ko (delta 0.37Ko) gzipped